spyder_terminal/widgets/terminalgui.py

```diff
@@ -235,35 +238,6 @@ def wheelEvent(self, event):
         delta = event.angleDelta().y()
         self.eval_javascript('scrollTerm({0})'.format(delta))
 
-    def event(self, event):
```

Comment:
Copy and paste are not working in this PR, but they work in master. So perhaps you don't need to remove this method?

However, opening a new terminal is working as expected.